packages/frontend/src/stores/history.ts

```diff
@@ -0,0 +1,11 @@
+import type { Readable } from 'svelte/store';
+import { Messages } from '/@shared/src/messages/Messages';
+import { bootcClient } from '/@/api/client';
+import type { BootcBuildInfo } from '/@shared/src/models/bootc';
+import { RPCReadable } from '/@/stores/rpcReadable';
+
+export const historyInfo: Readable<BootcBuildInfo[]> = RPCReadable<BootcBuildInfo[]>(
```

Comment:
Could you rename the file to historyInfo.ts? That would both match the const and filename, and avoid accidentally importing or talking about the 'wrong' history.ts.

Reply:
Done in new commit.